libcudacxx/include/cuda/std/detail/libcxx/include/utility

```diff
@@ -118,15 +118,14 @@ struct common_type<pair<T1, T2>, pair<U1, U2>>;
 template<class T1, class T2> pair(T1, T2) -> pair<T1, T2>;
 
 template <class T1, class T2> bool operator==(const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14
-template <class T1, class T2> bool operator!=(const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14, removed in C++20
-template <class T1, class T2> bool operator< (const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14, removed in C++20
-template <class T1, class T2> bool operator> (const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14, removed in C++20
-template <class T1, class T2> bool operator>=(const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14, removed in C++20
-template <class T1, class T2> bool operator<=(const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14, removed in C++20
-template <class T1, class T2>
-  constexpr common_comparison_type_t<synth-three-way-result<T1>,
-                                     synth-three-way-result<T2>>
-    operator<=>(const pair<T1,T2>&, const pair<T1,T2>&);                               // C++20
+template <class T1, class T2> bool operator!=(const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14, removed in
+C++20 template <class T1, class T2> bool operator< (const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in C++14,
+removed in C++20 template <class T1, class T2> bool operator> (const pair<T1,T2>&, const pair<T1,T2>&); // constexpr in
+C++14, removed in C++20 template <class T1, class T2> bool operator>=(const pair<T1,T2>&, const pair<T1,T2>&); //
+constexpr in C++14, removed in C++20 template <class T1, class T2> bool operator<=(const pair<T1,T2>&, const
+pair<T1,T2>&); // constexpr in C++14, removed in C++20 template <class T1, class T2> constexpr
+common_comparison_type_t<synth-three-way-result<T1>, synth-three-way-result<T2>> operator<=>(const pair<T1,T2>&, const
+pair<T1,T2>&);                               // C++20
```

Comment:
This got mangled.

Reply:
I dropped the whole synposis. No one reads it and it is heavily out of date all the time